ui/components/institutional/jwt-url-form/jwt-url-form.js

```diff
@@ -69,45 +70,53 @@ const JwtUrlForm = (props) => {
         {(!showJwtDropdown || addNewTokenClicked) && (
           <Box width={BlockSize.Full}>
             <Text
-              className="jwt-url-form__instruction"
               display={Display.Block}
               variant={TextVariant.bodyMd}
               marginBottom={4}
             >
               {props.jwtInputText}
             </Text>
 
-            <textarea
-              className="jwt-url-form__input-jwt"
+            <TextArea
               data-testid="jwt-input"
               id="jwt-box"
               onChange={(e) => {
                 props.onJwtChange(e.target.value);
               }}
               value={props.currentJwt}
               autoFocus
+              height="154px"
+              resize="both"
+              boxProps={{
+                borderRadius: 'sm',
+                borderStyle: 'solid',
+                backgroundColor: Color.backgroundDefault,
+                color: Color.textDefault,
```

Comment:
```suggestion
                backgroundColor: Background.backgroundDefault,
                color: TextColor.textDefault,
```

and you can find them in `ui/helpers/constants/design-system.ts` :)

Reply:
🥇